.github/actions/terraform-version/action.yml

```diff
@@ -0,0 +1,64 @@
+name: 'Terraform Version Action'
+description: 'Determine the Terraform version to use'
+inputs:
+  directory:
+    description: 'Path to Terraform Directory'
+    required: true
+  version:
+    description: 'Terraform version, nb: Setting this, overrides the version defined in code. If not set in code, the latest version is used.'
+    required: true
+
+outputs:
+  version:
+    description: 'Terraform version'
+    value: ${{ steps.set-terraform-version.outputs.version }}
+
+runs:
+  using: 'composite'
+  steps:
+    - name: Set Terraform version
+      id: set-terraform-version
+      env:
+        inputs_version: ${{ inputs.version }}
+      working-directory: ${{ inputs.directory }}
+      shell: bash
+      run: |
+        terraform=version
```

Comment:
```suggestion
```

Reply:
![image](https://github.com/user-attachments/assets/222e1727-36de-4086-b3ef-3920f9adaa21)
det er for at det skal se "fint ut" her